code/nrf-connect/prstlib/boards/arm/bparasite_nrf52840/Kconfig

```diff
@@ -11,4 +11,8 @@ config BOARD_ENABLE_DCDC_HV
 	select SOC_DCDC_NRF52X_HV
 	default y
 
-endif  # BOARD_BPARASITE_NRF52840
\ No newline at end of file
+endif  # BOARD_BPARASITE_NRF52840
+
+config CORE_LOG_LEVEL
```

Comment:
I don't know about the enum values but would guess not because the Kconfig is resolved first before any C code is compiled. But I did not research this further...

I thought about the config hierarchy and did not come up with a nice solution without duplicating code, since there is no real common ancestor in the device tree. I don't know if it is maybe possible to have more than one ancestor (like a mixin) and maybe have a common `b-parasite` mixin with common configuration flags for both MCU variants. I would really like that as it might help to reduce configuration errors and feature drifts...

Reply:
I did a quick test:

```
config PRSTLIB_DEFAULT_LOG_LEVEL
	int "Default log level for prstlib"
	default LOG_LEVEL_INF
```
And unfortunately that doesn't work :( (`LOG_LEVEL_INF` is undefined at this point). It seems like Zephyr implements the following strategy for subsystem logging:

<img width="539" alt="image" src="https://user-images.githubusercontent.com/1573409/208300481-5a312b0c-7ea2-4340-aa88-3abb64ae7ae2.png">

Each subsystem defines their own choices. I'm not sure if this is overkill for us, but it would perfect if we could just add a `Kconfig` file to `prstlib` and do it there.

Reply:
Changed the naming of the config flag as discussed.